spark/scala-2.11/pom.xml

```diff
@@ -33,7 +33,7 @@
   <name>Zeppelin: Spark Interpreter Scala_2.11</name>
 
   <properties>
-    <scala.version>2.11.8</scala.version>
+    <scala.version>2.11.12</scala.version>
```

Comment:
why this? spark is still on 2.11.8 https://github.com/apache/spark/blob/master/pom.xml#L158

Reply:
I see, you are right. I though we could have used some fixes between the version but now I see, this may cause incompatibility.